### PR TITLE
Fix builds: changelog inclusions fails on CI

### PR DESCRIPTION
**Pull request by khawkins98**
Some remote CI environments unhappy (gitlab, netlify) after the addition of `{% include 'changelog.md' %}. #1341

Local and github actions are happy. 

All are on node 12.

_computers_

### CODE REVIEW

**Review by sturobson — APPROVED** [review]

File: tools/vf-component-library/src/site/_includes/layouts/components.njk [hunk 1]
```diff
@@ -250,7 +250,7 @@ import { {{ component.baseHandle | camelize(true) }} } from "@visual-framework/{
       <details class="vf-details" id="changelog">
         <summary class="vf-details--summary">Changelog</summary>
 
-        {% include 'changelog.md' %}
+        {# {% include 'changelog.md' %} #}
```
Comment (sturobson) [inline tools/vf-component-library/src/site/_includes/layouts/components.njk]: Is this commented out now?

**Comment by khawkins98** [comment]
Now working.
https://app.netlify.com/sites/ecstatic-noether-150ab4/deploys/6017ed710928610007a37b9e

The `{% render %}` in a changelog was the issue.

https://github.com/visual-framework/vf-core/pull/1342/files#diff-f304fc54f10e04631189a5589f61bf69f1a63a676e40ff6449b7c27846bcd358L3

I'm not sure why it was parsing this on CI but no local — computers. Might be a node version thing.

In any case, have removed it.

Also removed the lock file for best practice and an unneeded line of code.

**Comment by khawkins98** [comment]
Spoke too soon, failing again.

File: tools/vf-component-library/src/site/_includes/layouts/components.njk (hunk 1 above)
Comment (khawkins98) [reply tools/vf-component-library/src/site/_includes/layouts/components.njk]: Just a bit of testing.

**Comment by khawkins98** [comment]
Ok, now it's working: https://app.netlify.com/sites/ecstatic-noether-150ab4/deploys/6017f0822a04460007f639b4

Seems that `ignore missing` and case sensitivity helped. 

https://github.com/visual-framework/vf-core/pull/1342/commits/a2f675deec696f900206b47f110ecfd0993a2dc9

**Pull request merged**